lib/autoyast.rb

```diff
@@ -23,6 +23,15 @@ def initialize(description)
       "repositories",
       "packages"
     )
+    if !description.users
+      Machinery::Ui.puts(
+        "\nWarning: Exporting a description without the scope 'users' as AutoYaST" \
+        " profile will result in a root account without a password which prevents" \
+        " logging in.\n" \
+        "So either inspect the scope 'users' before the export or" \
```

Comment:
I have talked to Cornelius about this. I would have required the users scope for exporting Autoyast, similar to packages and repositories but this would have force the user to add this section manually to the manifest if he created the file on its own.
That's why we decided to add the warning to be not that restrictive.